src/test/scala/tech/cryptonomic/conseil/tezos/InMemoryDatabase.scala

```diff
@@ -13,25 +13,28 @@ import scala.concurrent.duration._
 trait InMemoryDatabase extends BeforeAndAfterAll with BeforeAndAfterEach {
   self: TestSuite =>
 
-  /** defines configuration for a randomly named h2 in-memory instance */
-  protected val confString =
+  def inMemoryDbName: String
+
+  /** defines configuration for a custom-named H2 in-memory instance */
+  lazy protected val confString =
     s"""conseildb = {
-       |    url = "jdbc:h2:mem:conseil-test;MODE=PostgreSQL;DB_CLOSE_DELAY=-1"
+       |    url = "jdbc:h2:mem:$inMemoryDbName;MODE=PostgreSQL;DB_CLOSE_DELAY=-1"
```

Comment:
this change will probably conflict with the master

Reply:
I'll fix that when merging `master` then